ocp4-logging/.applier/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,83 @@
+---
+
+openshift_templates_raw: "https://raw.githubusercontent.com/redhat-cop/openshift-templates"
+openshift_templates_version: "v1.4.14"
+
+openshift_cluster_content:
+- galaxy_requirements:
+  - "{{ inventory_dir }}/../galaxy-requirements.yml"
+- object: Cluster Logging Stack
+  content:
+  - name: Create Namespace 
+    template: "{{ openshift_templates_raw }}/{{ openshift_templates_version }}/namespace/create-namespace.yml"
+    params_from_vars:
+      NAMESPACE: "openshift-logging"
+    post_steps:
+      - role: casl-ansible/roles/openshift-labels
+        vars:
+          label: "openshift.io/cluster-logging=true openshift.io/cluster-monitoring=true"
+          target_object: "namespace"
+          target_name: "openshift-logging"
```

Comment:
could the namespaces been parameterised?

Reply:
Switched to using params files for this. In the case of namespaces (in this instance), you really don't want to be changing most of the supplied parameters because they're set to the values of what they need to be in order for the operators to know what to do with them.

Reply:
ok, I think I understand.
You have to pass `NAMESPACE` to the namespace template but in this particular use case it has to be set to `openshift-logging` (I see it hardcoded further down).